server/chat-commands.js

```diff
@@ -2803,9 +2805,22 @@ const commands = {
 		let name = this.targetUsername;
 		if (!targetUser && !room.log.hasUsername(target)) return this.errorReply(`User ${target} not found or has no roomlogs.`);
 		let userid = toID(this.inputUsername);
-		if (!this.can('mute', null, room)) return;
 
-		if (targetUser && (cmd === 'hidealtstext' || cmd === 'hidetextalts' || cmd === 'hidealttext')) {
+		if (!user.can('mute', targetUser, room)) {
+			if (!this.can('mute', null, room)) return;
+			if (targetUser !== user) {
+				if (!cmd.includes('force')) {
+					return this.errorReply(`${target} is a higher rank than you, are you sure you want to hide their messages? Use /forcehidetext if you're sure.`);
```

Comment:
Why???

Why not just any auth? I don't see why relative rank should matter.

Reply:
Mirroring how every other punishment on auth works, mostly.

I wouldn't be opposed to a monitor for if an auth was punished in any capacity, but that's definitely something I'd want to put through policy staff first.

Reply:
That's not how `/force`-anything works. It's not a permission check, it's a typo guard.

Reply:
`/lock` vs `/forcelock` works the same way.

Reply:
Would you want the monitor to be on any use on auth, then?

Reply:
It depends on how common people use it on auth. I would exempt using it on themselves, but otherwise yes. We can update it if it's too spammy.

Reply:
Made a `trusted` check a la forcelock.